full/src/test/java/apoc/uuid/UUIDTest.java

```diff
@@ -62,6 +61,26 @@ public void testUUID() {
             tx.commit();
         }
     }
+    
+    @Test
+    public void testUUIDWithSetLabel() {
+        // given
+        db.executeTransactionally("CREATE CONSTRAINT ON (p:Mario) ASSERT p.uuid IS UNIQUE");
+        db.executeTransactionally("CALL apoc.uuid.install('Mario') YIELD label RETURN label");
+        // when
+        db.executeTransactionally("CREATE (p:Luigi {foo:'bar'}) SET p:Mario");
```

Comment:
No, this test worked even before, because it was managed by `txData.createdNodes();`.
I added it for coverage purpose and to compare it with the other case, `MATCH test (p: Peach) SET p: Mario`.